app/assets/stylesheets/custom.scss

```diff
@@ -137,3 +137,20 @@ a {
 td {
   background-color: rgba(255, 255, 255, .5);
 }
+
+// variables, mixins
+@mixin box_sizing {
```

Comment:
Name of mixin `box_sizing` should be written in all lowercase letters with hyphens instead of underscores